sinaloa/src/sinaloa_sgx.rs

```diff
@@ -33,14 +33,20 @@ pub mod sinaloa_sgx {
         sonora_sgx_ra_proc_msg2_trusted, sonora_start_local_attest_enc,
     };
     use std::{ffi::CStr, mem};
+    use std::io::Write;
+    use tempfile;
     use veracruz_utils;
 
     lazy_static! {
         static ref SONORA: std::sync::Mutex<Option<SgxEnclave>> = std::sync::Mutex::new(None);
     }
 
-    static MC_ENCLAVE_FILE: &'static str = "./target/debug/mexicocity.signed.so";
-    static SONORA_ENCLAVE_FILE: &'static str = "./target/debug/sonora.signed.so";
+    static MC_ENCLAVE_BINARY: &'static [u8] = include_bytes!(
```

Comment:
Note this change, I've embedded the MC/Sonora libraries into the binary so that it's possible to move the binaries without a relative path dependency. To pass to SGX I just write these to a temporary file when needed.

Reply:
This is interesting, and I like it. But would it be more appropriate to put it in a separate PR?

Reply:
Can do. Will keep this here but create an additional PR to bring in first, since it's needed for this PR.